studyzen/Lectures/LecturesController.cs

```diff
@@ -0,0 +1,73 @@
+using Microsoft.AspNetCore.Mvc;
+using Studyzen.Lectures;
+using StudyZen.Common;
+using StudyZen.Courses;
+using StudyZen.Lectures.Forms;
+using StudyZen.Lectures.Requests;
+
+namespace StudyZen.Lectures;
+
+[ApiController]
+[Route("lectures")]
+public sealed class LecturesController : ControllerBase
+{
+    private readonly ILectureService _lectureService;
+
+    public LecturesController(ILectureService courseService)
+    {
+        _lectureService = courseService;
+    }
+
+    [HttpGet]
+    [Route("{lectureId}")]
+    public async Task<IActionResult> GetLecture(int lectureId)
+    {
+        Lecture? requestedLecture = _lectureService.GetLectureById(lectureId);
+        if (requestedLecture == null)
+        {
+            return NotFound();
+        }
+        else
+        {
+            return Ok(requestedLecture);
+        }
+    }
+
+    [HttpPost]
+    public async Task<IActionResult> CreateLecture([FromForm] CreateLectureForm form, [FromBody] CreateLectureRequest? request)
```

Comment:
I get internal server error when using swagger. Fix it or lets remove `form `for now

Reply:
fixed, forms will be implemented later